lib/models.js

```diff
@@ -262,14 +254,16 @@ class Constraint {
   }
 }
 
-class Valueset extends DataElement {
+class Valueset  {
     constructor(identifier, isEntry=false) {
-        super(identifier, isEntry);
+        this._identifier = identifier;
         this._codes = [];
         this._rules = [];
         this._concepts = [];
     }
 
+    get identifier() { return this._identifier; }
+
```

Comment:
I apologize for missing this earlier, but typically valuesets are actually identified by a URL -- so I think we need a getter and setter for a `this._url` field -- and we need to set that `url` when we create the valueset.

To create it, I think we basically take the default path and add the valueset name at the end.  So, given this:
```
ValueSetDefinitions: shr.foo 
Path: http://standardhealthrecord.org/foo/vs/

ValueSet: Stuff
#abc
#def
```
The URL for the `Stuff` valueset should be `http://standardhealthrecord.org/foo/vs/Stuff`.  Make sense?